react/fields/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './Checkbox.less';
+import CheckMarkIcon from '../../icons/CheckMarkIcon/CheckMarkIcon';
+import classnames from 'classnames';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class Checkbox extends Component {
+  static displayName = 'Checkbox';
+
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    className: PropTypes.string,
```

Comment:
For props that are not required can you please include a `defaultProp`, makes the component's api more self-documenting